app/scripts/modules/amazon/src/function/configure/FunctionSettings.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { FormikFormField, IWizardPageComponent, TextInput, NumberInput, HelpField } from '@spinnaker/core';
+import { FormikProps } from 'formik';
+import { IAmazonFunctionUpsertCommand } from 'amazon/index';
+import { IAmazonFunction } from 'amazon/domain';
+
+export interface IFunctionSettingsProps {
+  formik: FormikProps<IAmazonFunctionUpsertCommand>;
+  isNew?: boolean;
+  functionDef: IAmazonFunction;
+}
+
+export class FunctionSettings extends React.Component<IFunctionSettingsProps>
```

Comment:
This can be a functional component.

Reply:
It needs to implement the IWizardPageComponent interface. 
Can I still make it a functional component?

Reply:
I am actually not sure